addons/account/models/account_move.py

```diff
@@ -2233,8 +2233,9 @@ def copy(self, default=None):
         copied_am = super().copy(default)
         message_origin = '' if not copied_am.auto_post_origin_id else \
             (Markup('<br/>') + _('This recurring entry originated from %s')) % copied_am.auto_post_origin_id._get_html_link()
+        message_content = 'This entry has been reversed from %s' if default.get('move_type') == 'out_refund' else 'This entry has been duplicated from %s'
```

Comment:
Note that for the Misc entry, as we're in the copy function, you might need to use the field `reversed_entry_id` if we want to do so

Reply:
The fact that i only did it for invoice was a misunderstood on my part since the first bullet point was only mentioning invoices but yes i needed to do it for bills too. 

For Misc, Laura didn't thought about it but told me is a good idea and we will change a bit the spec of the task, thanks !